sohbaker/EchoServer_Java
# Amends

- ClientHandlerTest: convert field to local variable
- ensure no new line at end of files
- add space between 'import' and 'import' static declarations
- MessageHandler: rename method to printIOExceptionErrors (specify which exception type the method expects in the naming).
